superset/commands/utils.py

```diff
@@ -62,6 +62,25 @@ def populate_owners(
     return owners
 
 
+def update_owner_list(
```

Comment:
nit: this is a pure function, but the name implies it's updating something but it's not, since it returns a newly created `list[User]`. why not call it `generate_owners` or `compute_owners`

Reply:
that makes sense! Updated the functions to `compute_owners` (in the class), and `compute_owner_list` (in `utils`).